### PR TITLE
Add readme documentation

### PR DESCRIPTION
**Pull request by Gredja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
 # Query Utils
 
 Query Utils extensions for IQueryable to perform Paging, Filer, Sorting
 
-## About
-
-This project is maintained by Softeq Development Corp.
+## Usage
+Query Utils has interfaces:
+- IPagedQuery - pagination results
```
Comment (eugepae) [inline README.md]: Need a more detailed description for every interface.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+- IFilteredQuery - filter results in query
+- ISortedQuery - sorting results in query
+
+Query class
+```csharp
+    public class GetProfilesQuery : IPagedQuery, IFilteredQuery, ISortedQuery
```
Comment (eugepae) [inline README.md]: I think, it's not important class for short readme

Comment (eugepae) [reply README.md]: Exchange it with filters and orders implementation sample.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,47 @@
...
 We specialize in .NET core applications.
 
+ - [Facebook](https://web.facebook.com/Softeq.by/)
+ - [Instagram](https://www.instagram.com/softeq/)
+ - [Twitter](https://twitter.com/Softeq)
+ - [Vk](https://vk.com/club21079655)
+
 ## Contributing
 
 We welcome any contributions.
```
Comment (eugepae) [inline README.md]: Please add additional section like other repositories has.

Comment (Gredja) [reply README.md]: Don't understand. Structure is the same as in other repositories.

File: README.md (hunk 1 above)
Comment (eugepae) [reply README.md]: Query Utils extension currently supports the following types of data manipulations, which you may implement in your query:
- IPagedQuery (allows you to get a sample of data spitted by pages)
...

**Pull request merged**